assets/css/general.css

```diff
@@ -38,44 +48,101 @@ h2 {
     margin-top: 10px;
 }
 
-/* Utilities */
+html[dir='ltr'] .d-flex .flex-wrap > *:not(*:last-child) { /* sass-lint:disable-line class-name-format */
```

Comment:
Instead of using these very generic selectors, is there no more specific class to style? This approach is brittle and hard to understand.

Also, these don't seem like general styles so they shouldn't be in `general.css`.

Reply:
There isnt any speific class. I will add it. and i saw css related with those elemnts lie in here so i thought it was general. Will change it. :)

Reply:
Done. :)